### PR TITLE
[DOC UPDATE] Wildcard Matching Documentation

### PR DESCRIPTION
**Pull request by kingswanwho**
# [DOC UPDATE] Wildcard Matching Documentation

## Description

Documented Wildcard Matching function with example.

## Documentation

Documented in:  
https://drill.apache.org/docs/querying-directories/

## Testing
UT reference to:
https://github.com/apache/drill/blob/4aefcef2b665c5737471664912a26ef6ed9a6cfc/exec/java-exec/src/test/java/org/apache/drill/exec/store/dfs/TestGlob.java

### CODE REVIEW

File: _docs/query-data/query-a-file-system/040-querying-directories.md [hunk 1]
```diff
@@ -152,3 +152,22 @@ Starting in Drill 1.16, Drill uses a Value operator instead of a Scan operator t
  
 You can use [query directory functions]({{site.baseurl}}/docs/query-directory-functions/) to restrict a query to one of a number of subdirectories and to prevent Drill from scanning all data in directories.
 
+##Querying Multiple Directories in One Query by Wildcard Matching
```
Comment (luocooong) [inline _docs/query-data/query-a-file-system/040-querying-directories.md:155]: Put a whitespace between the `##` and `Querying`.

**Review by luocooong — CHANGES_REQUESTED** [review]

**Review by dzamo — APPROVED** [review]

**Comment by dzamo** [comment]
Please rebase on apache:gh-pages

File: _docs/query-data/query-a-file-system/040-querying-directories.md (hunk 1 above)
Comment (kingswanwho) [reply _docs/query-data/query-a-file-system/040-querying-directories.md]: Hi cong, I did the change as you required. Please take a look

**Review by luocooong — APPROVED** [review]
LGTM. +1

**Pull request merged**